sdk/couchbase-core/src/agent_ops.rs

```diff
@@ -32,70 +32,67 @@ impl Agent {
         self.inner.bucket_features().await
     }
 
-    pub async fn upsert<'a>(&self, opts: UpsertOptions<'a>) -> Result<UpsertResult> {
+    pub async fn upsert(&self, opts: UpsertOptions<'_>) -> Result<UpsertResult> {
```

Comment:
Has there been a change to the lifetime notation? I'm guessing that this means the same as before?